packages/onchainkit/src/internal/components/TextInput.tsx

```diff
@@ -63,11 +63,14 @@ export const TextInput = forwardRef(
 
     return (
       <input
+        aria-disabled={disabled}
```

Comment:
nit: This shouldn't be necessary since inputs' disabled attribute is already accessible. This would only be needed for custom inputs (e.g. using divs or spans) or if the value was different than that of the input's disabled attribute